rnndb/g80_defs.xml

```diff
@@ -72,17 +72,122 @@ xsi:schemaLocation="http://nouveau.freedesktop.org/ rules-ng.xsd">
 </enum>
 
 <enum name="g80_msaa_mode" inline="yes">
-	<value value="0x00" name="MS1"/>
-	<value value="0x01" name="MS2"/>
-	<value value="0x02" name="MS4"/>
-	<value value="0x03" name="MS8"/>
-	<value value="0x04" name="MS8_ALT" varset="chipset" variants="GT215-"/>
-	<value value="0x05" name="MS2_ALT" varset="chipset" variants="GT215-"/>
-	<value value="0x06" name="UNK6" varset="chipset" variants="GF100-"/>
-	<value value="0x08" name="MS4_CS4"/>
-	<value value="0x09" name="MS4_CS12"/>
-	<value value="0x0a" name="MS8_CS8"/>
-	<value value="0x0b" name="MS8_CS24" varset="chipset" variants="GF100-"/>
+	<value value="0x00" name="1X1"/>
+	<value value="0x01" name="2X1"/>
+	<value value="0x02" name="2X2"/>
+	<value value="0x03" name="4X2"/>
+	<value value="0x04" name="4X2_D3D" varset="chipset" variants="GT215-"/>
+	<value value="0x05" name="2X1_D3D" varset="chipset" variants="GT215-"/>
+	<value value="0x06" name="4X4" varset="chipset" variants="GF100-"/>
+	<value value="0x08" name="2X2_VC_4"/>
+	<value value="0x09" name="2X2_VC_12"/>
+	<value value="0x0a" name="4X2_VC_8"/>
+	<value value="0x0b" name="4X2_VC_24" varset="chipset" variants="GF100-"/>
+</enum>
+
+<enum name="G80_TIC_SOURCE">
+	<value value="0" name="ZERO"/>
+	<value value="2" name="R"/>
+	<value value="3" name="G"/>
+	<value value="4" name="B"/>
+	<value value="5" name="A"/>
+	<value value="6" name="ONE_INT"/> <!-- 0x00000001 -->
+	<value value="7" name="ONE_FLOAT"/> <!-- 0x3f800000 -->
+</enum>
+
+<enum name="G80_TIC_TYPE">
+	<value value="1" name="SNORM"/>
+	<value value="2" name="UNORM"/>
+	<value value="3" name="SINT"/>
+	<value value="4" name="UINT"/>
+	<value value="5" name="SNORM_FORCE_FP16"/>
```

Comment:
The doc says "native 8/16bit snorm components forced to FP16 format". I have added that information, although the name already somehow contains it.